structures/User.ts

```diff
@@ -0,0 +1,21 @@
+import Base from './Base';
+
+export enum UserPremiumType {
+  NONE,
+  NITRO_CLASSIC,
+  NITRO,
+}
+
+export default interface User extends Base {
+  avatar?: string;
+  bot?: boolean;
+  discriminator?: string;
+  email?: string;
+  flags?: number;
+  locale?: string;
+  mfa_enabled?: boolean;
+  premium_type?: UserPremiumType;
+  public_flags?: number;
```

Comment:
Since we're doing that, probably add public flags as a constant with the value it can have.

Reply:
we might have it moved later but that will be done already 👍🏻